README.md

```diff
@@ -1,2 +1,27 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+likeliness = 0
+count = 0
+
+print 'please enter your word'
+read word
+character = char{n}
+compare char {0} # compares the first character of the input with the solution
+
+if char{0} = true then :
+  likeliness = likeliness + 1
+  count = count + 1
+else : 
+  Break
```

Comment:
`Break` usually denotes early exit from a loop, however we are not in a loop here. What does `break` mean here?